reascripts/ReaSpeech/source/ReaSpeechMain.lua

```diff
@@ -16,38 +16,25 @@ function ReaSpeechMain:main()
   ctx = ImGui.CreateContext(ReaSpeechUI.TITLE, ReaSpeechUI.config_flags())
   Fonts:load()
   app = ReaSpeechUI.new()
+  app:open()
```

Comment:
lol yeah it rang my bell when i saw that code in `ReaSpeechMain`/`ReaSpeechUI` mostly matched the parameters of what i was working on. i guess instead of a `ToolWindow` this is more generally a...`ManagedWindow` or something with that vibe? 🤔